draft-ietf-drip-arch.md

```diff
@@ -462,27 +473,27 @@ Certificate:
 
 DRIP Identity Management Entity (DIME):
 
-> An entity that performs functions similar to a domain registrar/registry. A DIME vets Claims and/or Evidence from a registrant and delivers back Endorsements and/or Certificates in response. It is a high-level entity in the DRIP registration/provisioning process that can hold the role of RAA or HDA.
+> An entity that performs functions similar to a domain registrar/registry. A DIME vets Claims and/or Evidence from a registrant and delivers back Endorsements and/or Certificates in response. It is a high-level entity in the DRIP registration/provisioning process that can hold the role of HDA, RAA or root of trust (likely the HHIT prefix owner and/or DNS apex owner) for DETs.
```

Comment:
```suggestion
> An entity that performs functions similar to a domain registrar/registry. A DIME vets Claims and/or Evidence from a registrant and delivers back Endorsements and/or Certificates in response. It is a high-level entity in the DRIP registration/provisioning process that can hold the role of HDA, RAA, or root of trust (e.g., the HHIT prefix owner or DNS apex owner) for DETs.
```

Reply:
In this case, it is not an example, but a clarification that the root _of trust_ for any HHIT hierarchy would naturally be the root _of that hierarchy_ in address space (the IPv6 prefix owner) or name space (the DNS apex owner), and further that those 2 roots would typically be the same. Perhaps rather than "likely" we should use another word, such as "typically"?

Reply:
"Typically" is better, thanks.